spec/factories/fitbit_sleeps.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :fitbit_sleep do
+    minutes_asleep  480
+    minutes_awake    10
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.